src/main/resources/crafter/studio/database/createDDL-derby.sql

```diff
@@ -113,3 +113,36 @@ CREATE TABLE CSTUDIO_OBJECTMETADATA (
   LAUNCHDATE TIMESTAMP,
   UNIQUE (SITE, PATH)
 ) ;
+
+CREATE TABLE CSTUDIO_USER
+(
+    USERNAME VARCHAR(255) PRIMARY KEY NOT NULL ,
+    PASSWORD VARCHAR(255) NOT NULL,
+    FIRSTNAME VARCHAR(255) NOT NULL,
+    LASTNAME VARCHAR(255) NOT NULL,
+    EMAIL VARCHAR(255) NOT NULL
+) ;
+
+INSERT INTO CSTUDIO_USER (USERNAME, PASSWORD, FIRSTNAME, LASTNAME, EMAIL)
+    VALUES ('admin', 'vTwNOJ8GJdyrP7rrvQnpwsd2hCV1xRrJdTX2sb51i+w=|R68ms0Od3AngQMdEeKY6lA==', 'admin', 'admin', 'dbrkic@rivetlogic.com') ;
```

Comment:
Please don't use this email address.  Shouldn't it be a fake address?  Why is it hard coded?